IOSAccessAssessment/ViewModel/UserStateViewModel.swift

```diff
@@ -0,0 +1,30 @@
+//
```

Comment:
Oof, too many folders, but I guess this makes sense with the MVVM architecture (although that's not something we are using throughout the app... maybe a second round of refactoring is in order once all the main objectives RAI program are done).